api/v6/report_server.thrift

```diff
@@ -200,10 +201,12 @@ typedef list<CommentData> CommentDataList
 
 /**
  * Members of this struct are interpreted in "AND" relation with each other.
+ * Between the members there is "OR" relation.
```

Comment:
I think this comment is confusing. Maybe reword to something like between the elements of the list.